packages/xo-web/src/common/report-bug-button.js

```diff
@@ -75,11 +75,18 @@ const reportOnSupportPanel = async ({
     ),
   ])
 
-  const res = await post(SUPPORT_PANEL_URL, formData)
-  if (res.status !== 200) {
-    throw new Error('cannot get the new ticket URL')
+  try {
+    const res = await timeout.call(post(SUPPORT_PANEL_URL, formData), 1e4)
+    if (res.status !== 200) {
+      throw new Error(`not a valid response status (${res.status})`)
+    }
+    open(await res.text())
+  } catch (error) {
+    logger.warn('cannot get the new ticket URL', { error })
+
+    const encodedTitle = encodeURIComponent(title == null ? '' : title)
+    open(`https://xen-orchestra.com/#!/member/support?title=${encodedTitle}`)
```

Comment:
It doesn't seem to let you see the creation form with an empty title.

Reply:
I created a ticket in `support panel`, as work around we can specify  a default value.